app/src/main/java/com/antailbaxt3r/collegemate/activities/LoginActivity.java

```diff
@@ -124,16 +128,20 @@ private void handleSignInResult(GoogleSignInResult result) {
         }
     }
 
-    private void firebaseAuthWithGoogle(AuthCredential credential) {
```

Comment:
This is how it is supposed to be done

Reply:
I have written the code according to Firebase documentation.